package.json

```diff
@@ -20,6 +20,8 @@
   "author": "GitHub",
   "license": "CC0-1.0",
   "devDependencies": {
```

Comment:
You should add `eslint` to the `devDependencies`

Reply:
fixed it, it was done, but got reverted when merged from master.